package.json

```
@@ -49,6 +49,7 @@
     "prepare": "husky install",
     "prepare-release-artifacts": "node ./scripts/prepare-release-artifacts.js",
     "pretest-e2e": "yarn ensure-deps",
+    "sanitize:mocha:results": "node ./scripts/sanitize-mocha-results",
```

runs without the extension but suggesting this to be most consistent with our current styling 
```suggestion
    "sanitize:mocha:results": "node ./scripts/sanitize-mocha-results.js",
```

---

updated [82b333b](https://github.com/cypress-io/cypress/pull/31264/commits/82b333b838eae2905e1783aa34f59fa7a66caae2)